crate/kmip/src/kmip/kmip_types.rs

```diff
@@ -830,7 +830,8 @@ pub struct Attributes {
     /// symmetric key, etc.) SHALL be set by the server when the object is
     /// created or registered and then SHALL NOT be changed or deleted before
     /// the object is destroyed.
-    pub object_type: ObjectType,
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
I have trouble finding in KMIP the reference where they say it is optional, is it from here https://docs.oasis-open.org/kmip/kmip-spec/v2.0/os/kmip-spec-v2.0-os.html#_Toc6497496?

Reply:
if you find this ref, we could add it next to the `object_type`

Reply:
I didn't find it either but for `Locate` it is specified that `Attributes` could be empty
![image](https://github.com/Cosmian/kms/assets/1770543/bee39eb0-f458-48f6-9098-a68f9dba521f)

Reply:
ok nevermind